_includes/scripts.html

```diff
@@ -1,6 +1,10 @@
 <!--JQuery-->
 <script type="text/javascript" src="{{ site.baseurl }}/js/jquery-3.2.1.js"></script>
 
+<!-- Lunr.js for search -->
+<script type="text/javascript" src="//unpkg.com/lunr/lunr.js"></script>
+<script defer type="text/javascript" src="{{ site.url }}{{ site.baseurl }}/js/lunr-feed.js?3"></script>
```

Comment:
You can remove the query parameter here - that's used to force browsers to fetch the script if you've updated it.